kyuubi-server/src/main/scala/org/apache/kyuubi/client/KyuubiSyncThriftClient.scala

```diff
@@ -141,9 +141,6 @@ class KyuubiSyncThriftClient private (
    * Lock every rpc call to send them sequentially
    */
   private def withLockAcquired[T](block: => T): T = Utils.withLockRequired(lock) {
-    if (engineConnectionClosed) {
```

Comment:
> !protocol.getTransport.isOpen

https://issues.apache.org/jira/browse/THRIFT-2530
TIOStreamTransport's isOpen() always be true